src/util/helpers.ts

```diff
@@ -28,6 +29,15 @@ export const formatDec = (num: number) => {
   return Number(num.toFixed(Config.DECIMAL)).toString();
 };
 
+export const toBitcoinAmount = (amount: number | string) => {
+  const txAmount = typeof amount !== 'string' ? amount.toString() : amount;
```

Comment:
maybe this can be shortened to 
```suggestion
  const txAmount = String(amount);
```